plugins/mcreator-core/variables/logic.json

```diff
@@ -0,0 +1,7 @@
+{
+	"name": "logic",
+	"dependencyType": "boolean",

```

Comment:
I don't think this is actually used anywhere, because I couldn't find any other instance of `dependencyType`